routes/web.php

```diff
@@ -73,6 +70,9 @@
         $router->delete('/user/social/delete/{id}', ['uses' => 'UserController@socialDelete']);
         $router->post('/user/deactivate', ['uses' => 'UserController@deactivateUser']);
         $router->post('add-folder','UploadController@addFolder');
+        $router->post('/edit-camp-newsfeed','NewsFeedController@editNewsFeed');
```

Comment:
Probably need to use Route::Resource to define the routes

Reply:
Discussed it over Skype.